stack.yaml

```diff
@@ -17,8 +17,7 @@
 #
 # resolver: ./custom-snapshot.yaml
 # resolver: https://example.com/snapshots/2018-01-01.yaml
-resolver:
-  url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/18/21.yaml
+resolver: lts-19.33 # GHC 9.0.2
```

Comment:
because this is the GHC version with a ready-to-use binary for Haskell Language Server. Since compiling HLS is painfully long tt felt like a natural upper bound.